capstone/src/main/webapp/statistics_script.js

```diff
@@ -0,0 +1,250 @@
+/* global google */
+
+google.charts.load('current', {packages: ['corechart']});
+google.charts.setOnLoadCallback(getSurveyResponses);
+
+async function getSurveyResponses() {
+  const response = await fetch('/load-data');
+  const list = await response.json(); // list of entities from datastore
+  const scores = [];
+  /* sentimentCount: Very positive, positive, neutral, negative, and very neg.*/
+  const sentimentCount = [0, 0, 0, 0, 0];
+  const refGender = {
+    'M': 0,
+    'F': 1,
+    'U': 2,
+  };
+  /* genderCount: Male, female, and unknown */
+  const genderCount = [0, 0, 0];
+  const refAge = {
+    '18-24': 0,
+    '25-34': 1,
+    '35-44': 2,
+    '45-54': 3,
+    '55-64': 4,
+    '65+': 5,
+    'Unknown': 6,
+  };
+  /* ageCount: 18-24, 25-34, 35-44, 45-54, 55-64, 65+, or Unknown */
+  const ageCount = [0, 0, 0, 0, 0, 0, 0];
```

Comment:
These data structures are kinda hard to work with. What you could do instead (and what I meant in my last review is this):

```
const sentimentCount = {
    'very postitive': 0,
    'positive': 0,
    'neutral': 0,
    'negative': 0,
    'very negative': 0,
};
```

That way when you increment counts, it's much easier to read than indices.
`sentimentCount['very positive'] += 1;` as opposed to `sentimentCount[0] += 1`

The same comment goes for these other arrays storing counts.

Reply:
Oh I see now, thank you for the clarification, I'll add that change now

Reply:
I would like to keep the data stored in a list though since I have my charts set up to read lists